api/client/client.go

```diff
@@ -22,8 +23,7 @@ import (
 
 // A server points to a single remote CFSSL instance.
 type server struct {
-	Address string
-	Port    int
+	URL string
```

Comment:
In the future, we might consider adding an optional `*tls.Config` parameter and setting an `*http.Client` with that config.
e.g.:
```Go
type server struct {
	URL string
	client *http.Client
}
func newTLSServer(addr string, config *tls.Config) (*server, error) {
	srv, err := newServer(addr)
	if err != nil {
		return nil, err
	}

	srv.client = &http.Client{Transport: &http.Transport{TLSClientConfig: config}}
	return srv, nil
}

NewTLSServer(addr, &tls.Config{RootCAs: rootPool})
```